.travis.yml

```diff
@@ -3,20 +3,26 @@ sudo: required
 before_script:
   - npm install -g bower polymer-cli@next
   - polymer install --variants
+  - sudo mv /usr/bin/google-chrome /usr/bin/google-chrome-old
+  - sudo mv /usr/bin/google-chrome-beta /usr/bin/google-chrome
```

Comment:
this is necessary. It seems as if travis has an older version of google chrome (55) and we want to test beta instead. We do not want to push a new version of WCT-local which depends on launchpad which does not support chrome beta, so the quickest and most-reversible fix is to change the travis.yml to rename beta to standard chrome